packages/react-native/Libraries/Network/RCTNetworkTask.mm

```diff
@@ -16,7 +16,7 @@ @implementation RCTNetworkTask {
   id<RCTURLRequestHandler> _handler;
   dispatch_queue_t _callbackQueue;
   std::mutex _mutex;
-
+  std::atomic<RCTNetworkTaskStatus> _atomicStatus;
```

Comment:
The compiler complained about `_status` and `self.status` not having the same types. Thus, I renamed the ivar, but implemented the objective-C property getter. Thoughts @javache ?